ui/imports/shared/popups/ModalPopup.qml

```diff
@@ -67,10 +67,11 @@ Popup {
                 objectName: "titleText"
                 anchors.verticalCenter: parent.verticalCenter
                 anchors.left: parent.left
+                anchors.right: parent.right
+                anchors.rightMargin: 44
                 font.bold: true
                 font.pixelSize: 17
                 height: visible ? 24 : 0
-                width: visible ? parent.width - 44 : 0
```

Comment:
@alexandraB99 It seems that this line belongs to you. It caused problems when the popup was open the second time - the title text disappeared. I decided to replace it with anchors. Do you think the change is safe ?

Reply:
@endulab should be fine. elide mode should work equally when anchors set instead of width.